src/main/java/proto/SensorData.java

```diff
@@ -5,20 +5,21 @@
 
 public final class SensorData {
   private SensorData() {}
-  public static void registerAllExtensions(
-      com.google.protobuf.ExtensionRegistryLite registry) {
-  }
 
-  public static void registerAllExtensions(
-      com.google.protobuf.ExtensionRegistry registry) {
-    registerAllExtensions(
-        (com.google.protobuf.ExtensionRegistryLite) registry);
+  public static void registerAllExtensions(com.google.protobuf.ExtensionRegistryLite registry) {}
```

Comment:
This file's changes seem unrelated, can you split into another PR?

Reply:
This is the result of mvn spotless:apply which Jie mentioned to pass the Java check. This issue arose because some of the code in master (apparently these lines) wasn't in the appropriate format. I think it's fine for everyone to run mvn spotless:apply and commit even if it changes this file multiple times as git will recognize the change as the same and we should likely diligently run the command anyways and that way all checks will pass so we can prevent accidentally putting code in master that is in the improper format.